js/src/components/wpcom-account-card/connect-wpcom-account-card.js

```diff
@@ -2,19 +2,25 @@
  * External dependencies
  */
 import { __ } from '@wordpress/i18n';
+import { addQueryArgs } from '@wordpress/url';
 
 /**
  * Internal dependencies
  */
+import { glaData } from '.~/constants';
+import { API_NAMESPACE } from '.~/data/constants';
 import AppButton from '.~/components/app-button';
 import AccountCard, { APPEARANCE } from '.~/components/account-card';
 import useDispatchCoreNotices from '.~/hooks/useDispatchCoreNotices';
 import useApiFetchCallback from '.~/hooks/useApiFetchCallback';
 
 const ConnectWPComAccountCard = () => {
 	const { createNotice } = useDispatchCoreNotices();
+
+	const next = glaData.mcSetupComplete ? 'reconnect' : 'setup-mc';
```

Comment:
💅 

Wondering about the naming of "next" which sounds a bit generic to me. But I guess it comes from backend implementation

Reply:
Adjusted in https://github.com/woocommerce/google-listings-and-ads/pull/1385/commits/cabe35a5033fec920e3d91bcd96c011569f9cc1a.